.github/workflows/test.yaml

```diff
@@ -30,23 +47,25 @@ jobs:
       with:
         path: /var/cache/apt
         key: var-cache-apt-{{ hashFiles('**/debian/control') }}
-    - name: Run "apt update"
```

Comment:
This was separate because we otherwise fail on stale image cache. I wonder why mvo removed this part.

Reply:
hmm yes I remember this breaking frequently I agree it should be added back